index.html

```diff
@@ -217,10 +217,60 @@ <h2 class="about-cart-title">Skills</h2>
                     </div>
                 </div>
             </section>
+
+<!-- -----------------------------Form Setion-------------------------------- -->
+
+            <section class="from-section">
+                <span class="form-header">
+                    I'm always interested in hearing about new projects,
+                    so if you'd like to chat please get in touch.
+                </span>
+                <form class="form-itself" action="https://formspree.io/f/{form_id}" method="post">
```

Comment:
- [ ] Kindly make sure the form spree service is used correctly. You are supposed to put your own `form_id`. In that way, you will be able to get the user inputs in your mail.
After creating an account, just follow the steps listed in the link I just forward down below :smiley: 
`Hint: You may consider checking` [here](https://formspree.io/html/#:~:text=your%20website%27s%20CSS-,How%20it%20works,-With%20a%20couple) how to use the form spree service.